plugins/module_utils/ldap.py

```diff
@@ -34,6 +34,7 @@ def gen_specs(**specs):
     specs.update({
         'bind_dn': dict(),
         'bind_pw': dict(default='', no_log=True),
+        'ca_cert_file': dict(type="str"),
```

Comment:
Please use single quotes (like the surrounding code). Also, if the above is `path`, this should be too:
```suggestion
        'ca_cert_file': dict(type='path'),
```